src/main/java/no/nav/veilarbvedtaksstotte/config/KafkaConsumerConfig.kt

```diff
@@ -60,7 +60,6 @@ class KafkaConsumerConfig {
         val aivenConsumerClient = KafkaConsumerClientBuilder.builder()
             .withProperties(environmentContext.aivenConsumerClientProperties)
             .withTopicConfigs(consumerAivenConfig.configs)
-            .withToggle { unleashService.isKafkaKonsumeringSkruddAv }
```

Comment:
I dont think we should remove this. How we can stop kafka consuming, if we want to change kafka offset?

Reply:
Fair enough, I will leave it.